libs/providers/ofrep-web/src/lib/model/ofrep-web-provider-options.ts

```diff
@@ -0,0 +1,6 @@
+import { FetchAPI } from '@openfeature/ofrep-core';
+export interface OfrepWebProviderOptions {
+  baseUrl: string;
+  pollInterval?: number; // in milliseconds
+  disablePolling?: boolean;
```

Comment:
Could `pollInterval` take a negative value to disable polling?

Reply:
Yes it is a good idea, I'll change that.